src/structs/quantization_tables.rs

```diff
@@ -44,6 +50,9 @@ impl QuantizationTables {
             self.quantization_table[i] = quantization_table[RASTER_TO_JPEG_ZIGZAG[i] as usize];
         }
 
+        self.quantization_table_transposed =
```

Comment:
Please add comment why it's stored as AlignedBlock. Also add comments in idct, Otherwise, it's not obvious and might be occasionally reverted.